src/codegen/mod.rs

```diff
@@ -4356,6 +4403,35 @@ mod utils {
         args
     }
 
+    pub fn fnsig_argument_identifiers(
```

Comment:
Could you document what this returns? The signature is not super-self-descriptive.